test/data/Author_YYYY_FirstTitleWord_Page_fignr_identifier.yaml

```diff
@@ -1,6 +1,5 @@
-# This file can exist with the "source" information only, the entry data exists should be marked as "False" 
-# to indcate that neither the metadata nor the actual data were digitized or provided.
-# use "null" to idicate that a value is not available in the literature
+# This file can exist with the "source" information only
+# Figure description is adapted based on SVG input
```

Comment:
I don't understand this intro anymore. Should we delete it from these samples?

Reply:
Can we discuss this file separately, since it has no consequences on this pull request?
The file structure and content really depend on the workflow. This was a second draft we created a while ago. 

Among other issues, I also don't understand why it is located in the test folder.